### PR TITLE
Add UI and logic for contact details

### PR DESCRIPTION
**Pull request by DavidBiddle**
#### What problem does the pull request solve?

Trello card: https://trello.com/c/7u2OgU0G/19-add-form-specific-contact-details-to-forms-runner-should-have

#### Checklist

- [x] I've used the pull request template
- [x] I've linked this PR to the relevant issue (if mission work)
- [x] I've written unit tests for these changes (if code change)
- [n/a] I've updated the documentation in (If any documentation requires updating)
    - [n/a] README.md
    - [n/a] Elsewhere (please link)

### CODE REVIEW

File: app/views/forms/page/show.html.erb [hunk 1]
```diff
@@ -15,5 +15,30 @@
...
+        <% end %>
+        <% if @support_details.url.present? && @support_details.url_text.present? %>
+          <h3 class="govuk-heading-s"><%= t("support_details.online") %></h3>
+          <p class="govuk-body">
+            <%= govuk_link_to t("support_details.link_text", link: @support_details.url_text), @support_details.url, target:"_blank", rel: "noopener noreferrer" %>
+          </p>
+        <% end %>
+      <% end %>
+    <% end %>
+
```
Comment (aliuk2012) [inline app/views/forms/page/show.html.erb:42]: Could we many abstract this out into its own component which we could easily test different data setups? Also as a component we can easily setup a render condition so the view doens't have to have a big chunky condition in it

**Review by aliuk2012 — COMMENTED** [review]
Overall it looks good.  Good to see a fly-by fixup of the linting issues. Could you also update this line so it picks up all our code 

https://github.com/alphagov/forms-runner/blob/main/.github/workflows/rubyonrails.yml#L50
Should be a case of just removing `app` from that command.

File: app/views/forms/page/show.html.erb (hunk 1 above)
Comment (DavidBiddle) [reply app/views/forms/page/show.html.erb:20]: Done! Good suggestion

**Comment by DavidBiddle** [comment]
@aliuk2012 I did the rubocop change you suggested but then reverted it - it caused a lot of extra linting errors which I didn't see all of locally, so it felt like something that should be tackled in a separate PR.

**Review by aliuk2012 — APPROVED** [review]

**Pull request merged**